app/scripts/modules/core/cluster/filter/clusterFilter.model.js

```diff
@@ -30,6 +31,78 @@ module.exports = angular
 
     filterModelService.configureFilterModel(this, filterModelConfig);
 
+    this.multiselectInstanceGroups = [];
+    this.multiselectInstancesStream = new rx.Subject();
```

Comment:
The use of rx.Subject is inherently ambiguous?  Is the intention/design for this to be both a Observer and an Observable.  Looking at the rest of the code it looks like it is used only as an Observable.